### PR TITLE
Simplified Sass architecture

### PR DESCRIPTION
**Pull request by damianmuti**
Folks, this is a big step in polishing the v2 of our framework.
I've simplified the structure to what I consider the best possible way. Feel free to drop ideas on how to polish the new structure even more. :)

### CODE REVIEW

File: assets/styles/styles.scss [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
-/// 
+///
+/// -> Libs
+/// - A collection of dependencies
+///
 /// -> Helpers
-/// - Collection of custom SASS Functions, Mixins, placeholders, extends and helper classes.
-/// 	This file is included before the core since core makes use of some functions and
+/// - Collection of custom SASS Functions, Mixins, Animations and helper classes.
+/// 	This files are included before the core since core makes use of some functions and
```
Comment (arzafran) [inline assets/styles/styles.scss]: Really minor: _this_ should be replaced with _these_ since its meaning is now plural.

File: package.json [hunk 2]
```diff
@@ -76,6 +76,7 @@
     "gulp-bower": "0.0.13",
     "gulp-clean-css": "~2.0.13",
     "gulp-concat": "~2.6.0",
+    "gulp-css-globbing": "^0.1.9",
```
Comment (arzafran) [inline package.json]: Treat carets in versioning with care, it means the left-most non-zero digit will be modified, so this won't be updated to 0.2.0 and if the package has a 1.x.y the x may be changed bringing major changes with it. Try to use ~ instead. it's safer.

Comment (damianmuti) [inline package.json]: Looks like carets are the default when you add and save a package from the console (--save-dev).
Thatnks for the info, buddy.

File: assets/styles/styles.scss (hunk 1 above)
Comment (damianmuti) [inline assets/styles/styles.scss]: Good catch!

**Pull request merged**